BitmovinConvivaAnalytics/Classes/ConvivaAnalytics.swift

```diff
@@ -34,6 +34,7 @@ public final class ConvivaAnalytics: NSObject {
     let playerHelper: BitmovinPlayerHelper
     // Workaround for player issue when onPlay is sent while player is stalled
     var isStalled: Bool = false
+    var isPlaybackStarted: Bool = false
```

Comment:
In the `MetadataBuilder` we already have the same concept but we called the property `var playbackStarted: Bool = false` there.

I would propose to use the same naming here 🙂 _And making it private_

```suggestion
    private var playbackStarted: Bool = false
```

Reply:
done